platforms/core-configuration/model-reflect/src/main/java/org/gradle/internal/reflect/annotations/impl/DefaultTypeAnnotationMetadataStore.java

```diff
@@ -558,6 +572,30 @@ private void processPropertyMethodAnnotations(Method method, Map<String, Propert
         for (Annotation annotation : annotations.values()) {
             metadataBuilder.declareAnnotation(annotation);
         }
+
+        validatePropertyAccessorDoesNotOverrideForbiddenToOverrideMethod(method, metadataBuilder);
+    }
+
+    private static final String PROPERTY_ACCESSOR_MUST_NOT_BE_OVERRIDDEN = "PROPERTY_ACCESSOR_MUST_NOT_BE_OVERRIDDEN";
```

Comment:
💅 Should this constant be at the top of the class?

Reply:
Oh, I see others constants are also close the code they are used. Then leave it as it is, let's not change this in this PR

Reply:
This class is in heavy need of a refactor.